### ISSUE TITLE
Draw training and validating loss

### ISSUE THREAD

**Issue by hoanghd99** [benihime91/pytorch_retinanet#8]
Hi benihime91, thanks you so much for this repo. I had tried it so had worked, now I want to draw the training loss, precision, recall, validation loss, how I can do this because I don't familar with Pytorch, thanks you !
